src/main/java/org/apache/flink/api/java/io/neo4j/Neo4jFormatBase.java

```diff
@@ -199,10 +198,10 @@ public T setReadTimeout(int readTimeout) {
      * Validates mandatory arguments.
      */
     protected void validate() {
-      if (Strings.isNullOrEmpty(restURI)) {
+      if (restURI.isEmpty() || restURI == null) {
```

Comment:
the `restURI == null` check should be in front. If it's `null`, the `.isEmpty()` call will throw a NPE